apps/frontend/src/app/PageTeam/TeamSettingElement.tsx

```diff
@@ -125,23 +137,19 @@ function TeamCharacterSelector({ teamId }: { teamId: string }) {
   const onSelect = (cKey: CharacterKey) => {
     if (charSelectIndex === undefined) return
 
+    // Make sure character exists
+    database.chars.getWithInitWeapon(cKey)
+
     const existingIndex = teamCharIds.findIndex(
       (teamCharId) => database.teamChars.get(teamCharId)?.key === cKey
     )
     if (existingIndex < 0) {
-      if (teamCharIds[charSelectIndex]) {
-        // Already have a teamChar at destination, prompt for deletion
-        if (
-          !window.confirm(
-            `Do you want to replace existing character with a new character? The loadouts and data (such as multi-opts) on this existing character will be deleted.`
-          )
-        )
-          return
-        // delete destination character
-        const existingTeamCharId = teamCharIds[charSelectIndex]
-        database.teamChars.remove(existingTeamCharId)
-      }
-      const teamCharId = database.teamChars.new(cKey)
+      //find the first available teamchar
+      let teamCharId = database.teamChars.keys.find(
```

Comment:
iterate over `data` not `keys`?

Reply:
seems more complicated to:
```
let teamCharId = Object.entries(database.teamChars.data).find(([id,{key]=>key===cKey))?.[0]
```
Seems not as ergonomic